recipe/bld.bat

```diff
@@ -0,0 +1,32 @@
+
+mkdir build
+cd build
+
+set CMAKE_CONFIG="Release"
+
+cmake -LAH -G"NMake Makefiles"                ^
+  -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%"      ^
+  -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%"   ^
+  -DWITH_CGAL_ImageIO=OFF -DWITH_CGAL_Qt5=OFF ^
+  ..
+if errorlevel 1 exit 1
+cmake --build . --config %CMAKE_CONFIG% --target INSTALL
+if errorlevel 1 exit 1
+
+:: language bindings are in a separate repo without releases
+git clone https://github.com/CGAL/cgal-swig-bindings.git
+cd cgal-swig-bindings
+
+:: this test requires numpy and we do not want to build-depend on it
+del examples/python/test_aabb2.py
+
+cmake -LAH -G"NMake Makefiles"                ^
+  -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%"      ^
+  -DCMAKE_INSTALL_PREFIX="%PREFIX%"           ^
```

Comment:
in fact this needs https://github.com/CGAL/cgal-swig-bindings/pull/79 for everything to be installed correctly as there's a dll and some python bindings. you're right for the dll, but not for the bindings. my PR will allow to set CMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" while overriding python install dir.